Final/index.html

```diff
@@ -4,7 +4,7 @@
 <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Dino game</title>
```

Comment:
Please create a new div at the end and add your name as `<span> Your name </span>`
Something like this `<div> <span> Your name </span>  </div>`